### DISCUSSION TITLE
Favorites & FTB Buttons.

### DISCUSSION THREAD

**Discussion by SSkizz** [shedaniel/RoughlyEnoughItems#1275] [Q & A] [answered]
Looked around in the cfg, in discord, and here but there doesn't seem to be any mention on how to "separate" favorited items from the FTB buttons. I'm fairly sure I was watching a let's play of a modpack that was using REI, and they had a divider between the two. Is this possible with REI or was that perhaps a plugin?

**Comment by Shadowdragon019** [comment]
Who where you watching?
That sounds like JEI but I'm not sure.

**Reply by SSkizz** [reply]
I wish I could find the video, but I'm almost certain it was in Stoneblock 3, which uses REI.

**Comment by shedaniel** [comment, marked as answer]
It was a previous REI feature but removed, a similar feature will be added in the future

**Reply by SSkizz** [reply]
Cool, thanks for the info! Thanks for REI btw, I've come to love it quite quickly.
